### ISSUE TITLE
Small 5 Min Quickstart issue?

### ISSUE THREAD

**Issue by WonSong** [angular/angular.io#128]
In the Import Angular section in https://angular.io/docs/js/latest/quickstart.html... 

Shouldn't 

```
<reference path="typings/angular2/angular2.d.ts" >
```

be 

```
/// <reference path="typings/angular2/angular2.d.ts" />
```

Instead?

I am getting typescript compiler error using 1.5.0 beta without ```///``` and the enclosing slash ```/>```

**Comment by im-tollu** [comment]
It should, according to [http://www.typescriptlang.org/Handbook#modules-splitting-across-files]

**Comment by WonSong** [comment]
Closed by #129
